docs/guides/crud-actions.md

```diff
@@ -154,13 +153,28 @@ store.dispatch({
 });
 ```
 
-Later, when you dispatch the success action, the resources returned will be
-associated with this label. This will allow you to keep track of which books were
-returned by this specific search.
+Resources are primarily used to keep
+[track of the request status](/docs/guides/tracking-requests.md). For more, refer
+to [the guide on labels](/docs/guides/request-labels.md).
 
-Labels are one of the more difficult features of Resourceful Redux to wrap your
-head around, so there is [an entire guide](/docs/guides/request-labels.md) dedicated
-to explaining how to use them, and when to use them.
+### `list`
```

Comment:
prior doc references call this `lists` - is it supposed to be the same?

Reply:
This references is the actual name of the attribute on the action, rather than the concept in general, so it’s `list`. `label` above is similar (rather than “Labels”)